evap/student/templates/student_vote.html

```diff
@@ -14,8 +14,8 @@
 
 {% block content %}
     {{ block.super }}
-    {% if errors %}
-        <div class="alert alert-danger" role="alert">{% blocktrans %}Please make sure to vote for all contributors or click on "I can't give feedback" to skip a person.{% endblocktrans %}</div>
+    {% if is_bound %}
```

Comment:
regarding the other template parameters, course_forms and contributor_questionnaires sound like different things but the former is just the general contribution, right? could you check that and then call them [course/contributor]_forms? or maybe [course/contributor]_questionnaires, not sure which one is more accurate.